www-client/seamonkey/seamonkey-2.53.18.2-r1.ebuild

```diff
@@ -0,0 +1,568 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+
+PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_REQ_USE='ncurses,sqlite,ssl,threads(+)'
+
+LLVM_MAX_SLOT=18
+
+# This list can be updated with scripts/get_langs.sh from the mozilla overlay
+# note - could not roll langpacks for: ca fi
+#MOZ_LANGS=(ca cs de en-GB es-AR es-ES fi fr gl hu it ja lt nb-NO nl pl pt-PT
+#	    ru sk sv-SE tr uk zh-CN zh-TW)
+MOZ_LANGS=(cs de en-GB es-AR es-ES fr hu it ja lt nl pl pt-PT
+	    ru sk sv-SE zh-CN zh-TW)
+
+MOZ_PV="${PV}"
+MOZ_PV="${MOZ_PV/_beta/b}"
+MOZ_P="${P}"
+MY_MOZ_P="${PN}-${MOZ_PV}"
+PATCH_PV="2.53.18.2-r1"
+PATCH="${PN}-${PATCH_PV}-patches"
+PATCH_S="${WORKDIR}/gentoo-${PN}-patches-${PATCH_PV}"
+
+SRC_URI="https://archive.seamonkey-project.org/releases/${MOZ_PV}/source/${MY_MOZ_P}.source.tar.xz -> ${P}.source.tar.xz
+	https://archive.seamonkey-project.org/releases/${MOZ_PV}/source/${MY_MOZ_P}.source-l10n.tar.xz -> ${P}.source-l10n.tar.xz
+	https://github.com/BioMike/gentoo-${PN}-patches/archive/refs/tags/${PATCH_PV}.tar.gz -> ${PATCH}.tar.gz"
+
+S="${WORKDIR}/${MY_MOZ_P}"
+
+MOZ_GENERATE_LANGPACKS=1
+MOZ_L10N_SOURCEDIR="${S}/${P}-l10n"
+inherit autotools check-reqs desktop edos2unix flag-o-matic llvm mozcoreconf-v6 mozlinguas-v2 pax-utils \
```

Comment:
Could you migrate to *llvm-r1.eclass*?

Reply:
I've just committed this migration, I hope I did it correctly (a test build did work).